package.json

```
@@ -41,7 +41,8 @@
     "sideEffects": false,
     "files": [
         "lib/",
-        "src/"
+        "src/",
+        "umd/"
```

By convention, this directory would be better named `dist/`
```suggestion
        "dist/"
```
That allows us to reuse the dir in future for other bundle formats, e.g. ESM, CJS, etc. Or even include variants for different client target ECMAscript versions.